README.md

```diff
@@ -1,2 +1,10 @@
-# helm-charts
-Helm charts for StackRox Kubernetes Security Platform
+# Helm charts for the StackRox Kubernetes Security Platform
+
+To use Helm charts to install the [StackRox Kubernetes Security Platform](https://www.stackrox.com/platform/)
+follow the instructions in the `README.md` file under the corresponding version
+directory.
+
```

Comment:
```suggestion
After you [install StackRox Central](https://help.stackrox.com/docs/get-started/quick-start/#install-stackrox-central),
you can use helm charts to install Sensor, Collector, and Admission Controller. 

> **IMPORTANT**
>
> We publish new Helm charts with every new release of the StackRox
> Kubernetes Security Platform. Make sure to use a version that matches the
> version of the StackRox Kubernetes Security Platform you've installed.

For example, to install  Sensor, Collector, and Admission Controller using Helm
charts for the StackRox Kubernetes Security Platform version 3.0.41.4, view the
instructions in the
[`3.0.41.4` directory](https://github.com/stackrox/helm-charts/tree/master/3.0.41.4).

```

Reply:
link to the README instead of the directory in which it exists? ^ @gaurav-nelson

Reply:
Thanks @clickboo 

I was thinking that the README always shows up when we open the directory on GitHub, but yes we can directly link to README as well.

Reply:
It makes to link the directory, as it's where user will start exploring stuff if they're using GitHub web UI. And GitHub web UI will immediately show README per Gaurav's point. So let's keep it this way.